app/pb_kits/playbook/pb_typeahead/components/MultiValue.jsx

```diff
@@ -0,0 +1,45 @@
+/* @flow */
+
+import React from 'react'
+import { components } from 'react-select'
+
+import { FormPill } from '../../'
+
+type Props = {
+  data: object,
+  multiValueTemplate: any,
+  removeProps: any,
+}
+
+const MultiValue = (props: Props) => {
+  const {
+    data,
+    removeProps,
+  } = props
+
+  const handleOnClick = removeProps.onClick
+
+  return (
+    <components.MultiValueContainer
+        className="text_input_multivalue_container"
+        {...props}
+    >
+      <If condition={data.avatarUrl}>
+        <FormPill
+            avatarUrl={data.avatarUrl}
```

Comment:
In case renaming the prop is an option:

```suggestion
            imageUrl={data.imageUrl}
```